src/sql/parts/modelComponents/queryTextEditor.ts

```diff
@@ -129,13 +134,28 @@ export class QueryTextEditor extends BaseTextEditor {
 		// number of lines that wrap). Finally, viewportColumn is calculated on editor resizing automatically; we can use it to ensure
 		// that the viewportColumn will always be greater than any character's column in an editor.
 		let numberWrappedLines = 0;
-		for (let line = 1; line <= lineCount; line++) {
-			if (editorWidgetModel.getLineMaxColumn(line) >= this._config.editor.layoutInfo.viewportColumn - 1) {
-				numberWrappedLines += Math.ceil(editorWidgetModel.getLineMaxColumn(line) / this._config.editor.layoutInfo.viewportColumn);
+		let shouldAddHorizontalScrollbarHeight = false;
+		if (!this._editorWorkspaceConfig) {
+			this._editorWorkspaceConfig = this.workspaceConfigurationService.getValue('editor');
+		}
+		let wordWrapEnabled: boolean = this._editorWorkspaceConfig && this._editorWorkspaceConfig['wordWrap'] && this._editorWorkspaceConfig['wordWrap'] === 'on' ? true : false;
```

Comment:
As mentioned offline since you're worried about this method being called too often look at using `debounceEvent` to rate limite the times it's called.